### PR TITLE
Lionel Smith-Gordon Use of Asset in AssetCriteria

### PR DESCRIPTION
**Pull request by regnosys-prod-user**

### CODE REVIEW

**Comment by Oblongs** [comment]
Reviewed in the Collateral Working Group on 29 October 2024, see #3169.

File: docs/eligible-collateral-representation.md [hunk 1]
```diff
@@ -220,13 +220,13 @@ type AssetCriteria:
   agencyRating AgencyRatingCriteria (0..*)
   maturityType MaturityTypeEnum (0..1)
   maturityRange PeriodRange (0..1)
-  assetIdentifier AssetIdentifier (0..*)
+  specificAssets Asset (0..*)
```
Comment (chrisisla) [inline docs/eligible-collateral-representation.md:223]: The design principles impose a singular naming convention even when we are expecting lists of data i.e. `specificAsset` should be used here. I think adding the 's' on the end makes more sense in this instance.

**Review by chrisisla — DISMISSED** [review]

**Review by valdensmith — COMMENTED** [review]
Line 248, in the description, there is a typo on the word loan

**Review by valdensmith — COMMENTED** [review]
Everything else looks good as we discussed

**Review by valdensmith — APPROVED** [review]

**Comment by Oblongs** [comment]
> Same description is it also worth adding the details that this attribute can be used for specific source ids such as ISINS/ CUSIPS

Typo fixed and description enhanced

**Comment by valdensmith** [comment]
Something needs to be added to line 184 too for this

Vernon Alden-Smith
Director - Collateral Initiatives
ISDA

**Review by valdensmith — APPROVED** [review]

**Review by hugohills-regnosys — APPROVED** [review]

**Pull request merged**